README.md

```diff
@@ -16,3 +16,17 @@ Also give a theoretical argument for why X could or could not be correct, based
 on the complexity of the general sorting problem we covered in class.
 
 Add your answers to this markdown file.
+
+------------------------------------------------------------------------------
+
+I would start by running different sizes of array through the algorithm, so say that the I feed it an array of size 10 and it takes around 15 milliseconds to finish.  With this base line we can run a few more different sizes of array to find a trend.  Using a potential graph of $y=x+5$ So say that we then run an array of size 100 through it, and it takes around 105 milliseconds, and again on an array of size 1000 and this one takes around 1005 milliseconds, as we can see this way they show that it is supporting the researchers claim of $O(n)$ time.  But say that if we were doing the same method but had different results.  If we run an array of size 10 and still get 15 milliseconds as a start we can compare more arrays again, so if we do an array of size 100 but get 205 milliseconds, this change is noticeable but two points don't tell us much, so if we run an array of size 1000 into it and get around 3000 milliseconds we can tell that this is not increasing in a linear fashion, but in $O(nlogn)$ (times from $y=xlogx$) time because if it was linear the expected 3rd time on an array of 1000 would be around 2103 milliseconds with an equation I built using those two points of $y=2.11x-6.11$
+
+ I don't believe that this algorithm can be possible, adaptive sorting algorithms such as insertion sort can only achieve a complexity of $O(n)$ on arrays that are already sorted or extremely close to being sorted, so (1) if this algorithm is adaptive then the tests were insufficient or (2) if this algorithm isn't adaptive then it wouldn't even be able to reach a complexity that efficient (3) and finally according to Cormen et al in Introduction to Algorithms it's not possile for a comparison sort algorithm to not have a worst case of $\Omega(nlogn)$.
```

Comment:
And what about the theoretical complexity of the sorting problem that we talked about in class?

Reply:
I'm really sorry but could you remind me which one you're referring to?